src/create_wand_data.cpp

```diff
@@ -20,12 +21,15 @@ int main(int argc, const char **argv) {
     std::string output_filename;
     bool        variable_block = false;
     bool        compress       = false;
+    bool        range          = false;
 
     CLI::App app{"create_wand_data - a tool for creating additional data for query processing."};
     app.add_option("-c,--collection", input_basename, "Collection basename")->required();
     app.add_option("-o,--output", output_filename, "Output filename")->required();
     app.add_flag("--variable-block", variable_block, "Variable length blocks");
```

Comment:
We're not supporting variable blocks for doc ranges right now, are we? Maybe we should define variable block and doc range as mutually exclusive?

Reply:
yes!

Reply:
@YuhongZhang98 this needs to be done. You can read how to do it here: https://github.com/CLIUtils/CLI11/

Basic idea is simple: `add_option` returns a pointer to option that you later pass to a `excludes` method of another option; something like this:
```cpp
auto opt1 = app.add_option(...);
auto opt2 = app.add_option(...)->excludes(opt1);
opt1->excludes(opt2);
```

Reply:
Apparently only one `excludes` is enough https://github.com/CLIUtils/CLI11/pull/93/files